script.js

```diff
@@ -1,4 +1,119 @@
+var AWAITING_USER_NAME = "awaiting user name";
+var PSP_GAME = "psp game";
+var currentGameMode = AWAITING_USER_NAME;
+var userName = "";
+
+console.log("global");
+
+var winCount = 0;
+var loseCount = 0;
+var drawCount = 0;
+
+var totalCount = function (winCount, loseCount, drawCount) {
+  return Number(winCount) + Number(loseCount) + Number(drawCount);
+};
+console.log("stats");
+
+var winPercentage = function () {
+  return Math.floor((Number(winCount) * 100) / totalCount);
```

Comment:
Lastly, as you might realize after fixing the part about doing `winPercentage()` vs `winPercentage`, that executing `winPercentage` will always return `NaN`. The reason for that is `0 / 0` will be a "divide-by-zero" error, and JavaScript will just consider it `NaN`.

What you are missing here is that `winCount` and `totalCount` will always be 0, which is why you need a way to increase those numbers so they won't be stuck at 0.

This means making sure you increase `winCount` in the [winning if conditions](https://github.com/rocketacademy/basics-scissors-paper-stone/blob/eb62b2fe6fddbf4c1839fe01e02760f4cbe93eaa/script.js#L66-L84), increase `drawCount` in the [draw conditions](https://github.com/rocketacademy/basics-scissors-paper-stone/blob/eb62b2fe6fddbf4c1839fe01e02760f4cbe93eaa/script.js#L86-L100), and increase `loseCount` in the loss conditions (for your setup, probably need to create an if-block for loss, and inside the if-block all you are doing is increasing `loseCount` when the condition is met).